x-pack/plugins/security_solution/public/entity_analytics/components/risk_summary_flyout/risk_summary.stories.tsx

```diff
@@ -7,8 +7,7 @@
 
 import React from 'react';
 import type { Story } from '@storybook/react';
-import type { ExpandableFlyoutContextValue } from '@kbn/expandable-flyout/src/context';
-import { ExpandableFlyoutContext } from '@kbn/expandable-flyout/src/context';
+import { TestProvider } from '@kbn/expandable-flyout/src/test/provider';
```

Comment:
I understand that the import paths were defined before, but maybe now is a good time to change them to top-level imports instead of deep imports. Wdyt?

Reply:
so here it is a bit tricky - exposing test helpers like this is not recommended. the way to go about this is to put the entire provider into separate package and expose it that way it seems, but I dont want to add this in this PR specifically.